.eslintrc.json

```diff
@@ -29,6 +29,7 @@
         "no-extra-semi":"error",
         "no-extra-boolean-cast": "off",
         "react/jsx-uses-react": "error",
-        "react/jsx-uses-vars": "error"
+        "react/jsx-uses-vars": "error",
+        "no-console": "warn"
```

Comment:
Why did you decide to add this?

Reply:
In the project, there is ESLint and using console.log caused errors in it. Console logs were added with intentionally thus I think that they should not cause errors in ESLint.